lib/screens/explore_screens/regions_screens/regions_main/interactor/viewmodel/region_state.dart

```diff
@@ -28,21 +29,24 @@ class RegionState extends Equatable {
     RegionModel? region,
     bool? isBrowseView,
     TypeOfUsers? userType,
+    bool? loadingEvents,
   }) {
     return RegionState(
       pageCommand: pageCommand,
       pageState: pageState ?? this.pageState,
       region: region ?? this.region,
       isBrowseView: isBrowseView ?? this.isBrowseView,
       userType: userType ?? this.userType,
+      loadingEvents: loadingEvents ?? this.loadingEvents,
     );
   }
 
   factory RegionState.initial(RegionModel? region) {
     return RegionState(
-      pageState: PageState.initial,
+      pageState: PageState.success,
```

Comment:
? explain this change

Reply:
![image](https://user-images.githubusercontent.com/51983923/168911442-4a3058e4-7aa3-4097-a930-eec070613086.png)

pageState.Initial is not being use

Reply:
Is it a silent loading for the second screen?